### PR TITLE
Changes to block on demand run for multi config flow

### PR DESCRIPTION
**Pull request by prawalagarwal**

### CODE REVIEW

File: Tasks/VsTest/runvstest.ts [hunk 1]
```diff
@@ -21,6 +21,10 @@ if (osPlat !== 'win32') {
 
     try {
         utils.Helper.setConsoleCodePage();
+        const blockRun = isMultiConfigOnDemandRun();
+        if (blockRun) {
+            tl.setResult(tl.TaskResult.Failed, tl.loc('OnlyWindowsOsSupported'));
```
Comment (ShreyasRmsft) [inline Tasks/VsTest/runvstest.ts]: Change the message

File: Tasks/VsTest/runvstest.ts [hunk 2]
```diff
@@ -48,6 +52,17 @@ if (osPlat !== 'win32') {
     }
 }
 
+function isMultiConfigOnDemandRun(): boolean {
```
Comment (ShreyasRmsft) [inline Tasks/VsTest/runvstest.ts]: move to helper class

File: Tasks/VsTest/runvstest.ts (hunk 1 above)
Comment (ShreyasRmsft) [reply Tasks/VsTest/runvstest.ts]: to this MultiConfigNotSupportedWithOnDemand

File: Tasks/VsTest/runvstest.ts (hunk 2 above)
Comment (prawalagarwal) [reply Tasks/VsTest/runvstest.ts]: Why is that needed?

Comment (ShreyasRmsft) [reply Tasks/VsTest/runvstest.ts]: All such functions we have in that helper.ts file

Comment (prawalagarwal) [reply Tasks/VsTest/runvstest.ts]: This file is clean enough for now.

File: Tasks/VsTest/runvstest.ts (hunk 1 above)
Comment (prawalagarwal) [reply Tasks/VsTest/runvstest.ts]: Done. Thanks

**Comment by ShreyasRmsft** [comment]
Generate the resources resx file

**Pull request merged**